error.go

```diff
@@ -0,0 +1,19 @@
+package tprstorage
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microstorage"
+	"github.com/juju/errgo"
+)
+
+var invalidConfigError = microerror.New("invalid config")
+
+// IsInvalidConfig asserts invalidConfigError.
+func IsInvalidConfig(err error) bool {
+	return errgo.Cause(err) == invalidConfigError
+}
+
+// Errors from microstorage must be reused in order to match microstorage error
+// matchers. This is required to fulfil the interface and pass storagetest.
+
+var notFoundError = microstorage.NotFoundError
```

Comment:
This should only be a hack for some migration period. Reusing errors from other packages is wrong because it breaks the error API and leads potentially to weird and unexpected problems.